modules/ROOT/pages/audit-logging.adoc

```diff
@@ -675,6 +675,108 @@ Properties:
 * orgID
 * userID
 * branchName
+
+|Merge Branch
+|Project
+|Project ID
+|N/A
+|Merge Branch
+|Subaction:
+None +
+Properties:
+
+* orgID
+* userID
+* branchName
+
+|Share project
+|Project
+|Project ID
+|N/A
+|Share project
+|Subaction:
+None +
+Properties:
+
+* orgID
+* userID
+* projectName
+
+|Revert commit
+|Project
+|Project ID
+|N/A
+|Revert commit
+|Subaction:
+None +
+Properties:
+
+* orgID
+* userID
+* commitID
```

Comment:
We won't be including this at the end since the operation was not released into production. Can we remove this one please?